### PR TITLE
Improve branding

### PR DESCRIPTION
**Pull request by samuelmbabhazi**
## Description

Please include a summary of the changes and the related issue.

## Type of Change

-   [ ] Bug fix
-   [ ] New feature
-   [x] Breaking change
-   [ ] Documentation update

## Checklist

-   [x] My code follows the style guidelines of this project
-   [ ] I have performed a self-review of my code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have made corresponding changes to the documentation
-   [ ] My changes generate no new warnings

## Previous screenshots

Please add here videos or images of previous status

## Current screenshots

Please add here videos or images of previous status

### CODE REVIEW

File: apps/web/.env [hunk 1]
```diff
@@ -87,7 +87,13 @@ NEXT_PUBLIC_DISABLE_AUTO_REFRESH=false
 APP_NAME="Ever Teams"
 APP_SIGNATURE="Ever Teams"
 APP_LOGO_URL="https://app.ever.team/assets/ever-teams.png"
+NEXT_PUBLIC_APP_NAME="Ever Teams"
+NEXT_PUBLIC_COMPANY_NAME='Ever Co. LTD'
+NEXT_PUBLIC_LOGO_URL=
```
Comment (evereq) [inline apps/web/.env:92]: @samuelmbabhazi why you added NEXT_PUBLIC_LOGO_URL if we already had APP_LOGO_URL?
Same for NEXT_PUBLIC_APP_NAME while we had APP_NAME and so on!?

File: apps/web/components/layout/footer/footer.tsx [hunk 2]
```diff
@@ -14,7 +14,7 @@ const Footer = () => {
 						className="text-primary dark:text-gray-300"
 						rel="noreferrer"
 					>
-						{t('layout.footer.COPY_RIGHT4')}
+						{process.env.NEXT_PUBLIC_APP_NAME}
```
Comment (evereq) [inline apps/web/components/layout/footer/footer.tsx:17]: @samuelmbabhazi we should be using something from NextJs config, not use process.env in components directly!?

**Comment by evereq** [comment]
@samuelmbabhazi you still keep using `process.env.xxxx` instead of NextJs configs etc. @Cedric921 can you help to resolve this please so we merge it!?

**Comment by samuelmbabhazi** [comment]
> @samuelmbabhazi you still keep using `process.env.xxxx` instead of NextJs configs etc. @Cedric921 can you help to resolve this please so we merge it!?

@evereq , @Cedric921  I think this is good now !

File: apps/web/app/constants.ts [hunk 3]
```diff
@@ -77,6 +77,16 @@ export const APP_NAME = process.env.APP_NAME || 'Ever Teams';
 export const APP_SIGNATURE = process.env.APP_SIGNATURE || 'Ever Teams';
 export const APP_LOGO_URL = process.env.APP_LOGO_URL || 'https://app.ever.team/assets/ever-teams.png';
 export const APP_LINK = process.env.APP_LINK || 'https://ever.team/';
+export const APP_SLOGAN_TEXT = process.env.APP_SLOGAN_TEXT;
+
```
Comment (evereq) [inline apps/web/app/constants.ts:81]: @samuelmbabhazi @Cedric921 why no default values for all env vars??? What if someone have empty env file???

**Pull request merged**